tests/Tokenizer/TokenTest.php

```diff
@@ -511,6 +523,19 @@ public function provideToArrayCases(): \Generator
         ];
     }
 
+    public function testGetClassyTokenKinds(): void
+    {
+        static::assertSame([T_CLASS, T_TRAIT, T_INTERFACE], Token::getClassyTokenKinds(false));
+
+        if (\defined('T_ENUM')) {
+            static::assertSame([T_CLASS, T_TRAIT, T_INTERFACE, T_ENUM], Token::getClassyTokenKinds(true));
+        } else {
+            static::assertSame([T_CLASS, T_TRAIT, T_INTERFACE], Token::getClassyTokenKinds(true));
+        }
+
+        static::assertSame([T_CLASS, T_TRAIT, T_INTERFACE], Token::getClassyTokenKinds(false));
```

Comment:
isn't it the same as in line 528?

Reply:
it, the test is when called with different valued for the parameters after each other it still functions (since the array is part static and part not)